dockstore-integration-testing/src/test/java/io/dockstore/client/cli/EventResourceIT.java

```diff
@@ -0,0 +1,175 @@
+package io.dockstore.client.cli;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import io.dockstore.common.CommonTestUtilities;
+import io.dockstore.common.ConfidentialTest;
+import io.dockstore.common.Registry;
+import io.dockstore.common.ToolTest;
+import io.dockstore.webservice.jdbi.EventDAO;
+import io.dockstore.webservice.resources.EventSearchType;
+import io.swagger.client.ApiClient;
+import io.swagger.client.ApiException;
+import io.swagger.client.api.ContainersApi;
+import io.swagger.client.api.ContainertagsApi;
+import io.swagger.client.api.EventsApi;
+import io.swagger.client.model.DockstoreTool;
+import io.swagger.client.model.Event;
+import io.swagger.client.model.StarRequest;
+import io.swagger.client.model.Tag;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.contrib.java.lang.system.ExpectedSystemExit;
+import org.junit.contrib.java.lang.system.SystemErrRule;
+import org.junit.contrib.java.lang.system.SystemOutRule;
+import org.junit.experimental.categories.Category;
+
+import static io.swagger.client.model.DockstoreTool.ModeEnum.MANUAL_IMAGE_PATH;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * This test was originally in BasicIT, but it sometimes fails on travis and fails consistently locally if it is run with another test
```

Comment:
Make an issue if it hasn't been done already

Reply:
https://github.com/dockstore/dockstore/issues/3248